openwisp_utils/admin_theme/templates/admin/input_filter.html

```diff
@@ -0,0 +1,25 @@
+{% load i18n %}
+
+<div class="ow-filter ow-input-filter">
+  <div class="filter-title" title="{{ choice.display }}">
+    {% with choices.0 as all_choice %}
+    <h3>{% blocktrans with filter_title=title %} By {{ filter_title }} {% endblocktrans %}</h3>
+    <form method="GET" action="">
+      {% for key, value in all_choice.query_parts %}
+      <input type="hidden" name="{{ key }}" value="{{ value }}" />
+      {% endfor %}
+      <input type="text" class="ow-input-filter-field" title="{{ spec.value | default_if_none:'' }}"value="{{ spec.value | default_if_none:'' }}"
```

Comment:
a space is missing here before `value`.

I recommend breaking the attributes over new lines, eg:

```html
<input type="text"
       class="...."
       title="..."
       value="..."
/>
```